### PR TITLE
2111.1 Release Notes 

### PR DESCRIPTION
**Pull request by wesleymccollam**

### CODE REVIEW

File: docs/release-notes/relnotes-2111-1.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- **PingFederate**
+    - PingFederate 11.0.0 is available on [Dockerhub](https://hub.docker.com/r/pingidentity/pingfederate).
+
+### Enhancements
+
+- **Docker Images**
+    - Applied the log4j2 patch updated zip files to PingAccess and PingFederate per recommendation of the [Ping Identity CVE knowledge article](https://support.pingidentity.com/s/article/Log4j2-vulnerability-CVE-CVE-2021-44228).
+    - The applied patches are available on the Ping Identity CVE knowledge article.
+    - All images tagged with the sprint 2111.1 do not contain the Log4j2 vulnerability CVE-2021-44228.
+### Product Build Matrix
```
Comment (kenneth-ping) [inline docs/release-notes/relnotes-2111-1.md:25]: Does spacing look OK in the output?  I would have expected there would need to be some empty lines before the ### Product Build Matrix to make it looks consistent, but maybe not.

File: docs/release-notes/relnotes-2111-1.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- See the [Product Version, Image Release Matrix](../../reference/productVersionMatrix/)
```
Comment (kenneth-ping) [inline docs/release-notes/relnotes-2111-1.md:27]: Are you going to update the product matrix to reflect the new supported image?

File: docs/release-notes/relnotes-2111-1.md (hunk 1 above)
Comment (wesleymccollam) [reply docs/release-notes/relnotes-2111-1.md:25]: Spacing does appear to be good to go 👍

File: docs/release-notes/relnotes-2111-1.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (kenneth-ping) [inline docs/release-notes/relnotes-2111-1.md:1]: Why do we need a the relnotes-2111-1.md?  Is the currentRelease.md sufficient?

File: docs/release-notes/currentRelease.md [hunk 4]
```diff
@@ -1,60 +1,32 @@
 ---
-title: DevOps Docker Builds, Version 2111 (December 06 2021)
+title: DevOps Docker Builds, Version 2111.1 (December 16 2021)
 ---
-# Version 2111 Release Notes
+# Version 2111.1 Release Notes
 
-## DevOps Docker Builds, Version 2111 (December 06 2021)
+## DevOps Docker Builds, Version 2111.1 (December 16 2021)
```
Comment (kenneth-ping) [inline docs/release-notes/currentRelease.md:7]: I think we need some sort of note that tells customers about the purging of the affected images we are about to do and why we're doing it.  What do you think?

File: docs/release-notes/relnotes-2111-1.md (hunk 3 above)
Comment (wesleymccollam) [reply docs/release-notes/relnotes-2111-1.md:1]: This file is linked in the release [here](https://github.com/pingidentity/pingidentity-docker-builds/releases/tag/2111.1). currentRelease.md is also the same file, we're just linking them exactly to the release as well.

File: docs/release-notes/relnotes-2111-1.md (hunk 2 above)
Comment (wesleymccollam) [reply docs/release-notes/relnotes-2111-1.md:27]: Yes. Will do that now 🏃

File: docs/release-notes/relnotes-2111-1.md (hunk 1 above)
Comment (wesleymccollam) [reply docs/release-notes/relnotes-2111-1.md:25]: I decided to add this back for consistency.

File: docs/release-notes/currentRelease.md (hunk 4 above)
Comment (wesleymccollam) [reply docs/release-notes/currentRelease.md:7]: This has now been added. Good suggestion 😄

**Pull request merged**